Backends/PostScript/test/package.lisp

```diff
@@ -0,0 +1,31 @@
+;;;  (c) copyright 2019 Jan Moringen
+
+;;; This library is free software; you can redistribute it and/or
+;;; modify it under the terms of the GNU Library General Public
+;;; License as published by the Free Software Foundation; either
+;;; version 2 of the License, or (at your option) any later version.
+;;;
+;;; This library is distributed in the hope that it will be useful,
+;;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+;;; Library General Public License for more details.
+;;;
+;;; You should have received a copy of the GNU Library General Public
+;;; License along with this library; if not, write to the
+;;; Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+;;; Boston, MA  02111-1307  USA.
+
```

Comment:
This and the test added in ` rasterimage: add a basic smoke test ` should be unified into one system under `Tests/`. The test suite invocation should accept the output destination argument for running. Then the test op would be (if we must use uiop)
`(uiop:symbol-call '#:output-destination.test '#:run-tests "Postscript File" :file "foo.ps")` and similar for raster. Similar tests should be defined for PDF backend.

Reply:
All output destination tests now use the `print-test-page` function so the repeated code issue is gone. Tests for the PDF backend have been added.